apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityActions.tsx

```diff
@@ -88,40 +82,32 @@ const UtilityActions = ({
             type="text"
             onClick={() => prettifyQuery()}
             className="px-1"
-            icon={<IconAlignLeft size="tiny" strokeWidth={2} className="text-gray-1100" />}
+            icon={<AlignLeft size="tiny" strokeWidth={2} className="text-gray-1100" />}
```

Comment:
unsupported tailwind color

Reply:
probably text-foreground-muted, but now sure what it looks like

Reply:
i dont think we need that class name actually - will remove